src/main/java/com/metamx/common/io/smoosh/FileSmoosher.java

```diff
@@ -63,8 +72,13 @@
 
   private final List<File> outFiles = Lists.newArrayList();
   private final Map<String, Metadata> internalFiles = Maps.newTreeMap();
+  // list of files completed writing content using delegated smooshedWriter.
```

Comment:
I think it is better to write such comments as Javadoc `/** ... */`, then on automatic refactorings and reformattings IDEs will move comments along with fields. Javadoc comments are allowed for private fields.